omod/src/main/java/org/openmrs/module/patientflag_rest/web/wrapper/FlagWrapper.java

```diff
@@ -0,0 +1,48 @@
+package org.openmrs.module.patientflag_rest.web.wrapper;
+
+import java.io.Serializable;
+import java.util.List;
+
+import org.openmrs.BaseOpenmrsData;
+
+/**
+ * @author Stélio Moiane
+ *
+ */
+public class FlagWrapper extends BaseOpenmrsData implements Serializable {
```

Comment:
Do we definitely need this wrapper class? I'm not seeing something similar in other modules.